mobile/src/main/AndroidManifest.xml

```diff
@@ -114,5 +114,8 @@
         <!--    This is temp disabled because of a bug in GMS 6.5
                 <meta-data android:name="com.google.android.gms.analytics.globalConfigResource"
                     android:resource="@xml/global_tracker" /> -->
+
+        <meta-data android:name="com.google.android.geo.API_KEY"
+            android:value="AIzaSyBkvcuBAZXKfBZgpfCZji1Qbs49MiXm_Ws" />
```

Comment:
What's the best way to deal with this? Currently this is my API key, which obviously needs to change. Do we want to add a string resource for it? Or use a gradle variable populated by some (not checked in) gradle resources file?

Reply:
From https://developers.google.com/maps/documentation/android-api/signup?hl=en

> or Android apps, you restrict the key to your app's SHA-1 fingerprint and package name.

So I think @digitaldan can get a api key that only works for this app, so we could make it public.

Reply:
IIRC the manifest is merged from main + flavor. So please move the api key to mobile/src/full/AndroidManifest.xml

Reply:
Will do so (along with other full-only stuff, such as the GCM permissions) once I got the API key to use.

Reply:
I think you should do this in a new PR.

Reply:
As you prefer, I'm fine with it either way. I need the API key beforehand nevertheless, though ;-)

Reply:
> or Android apps, you restrict the key to your app's SHA-1 fingerprint and package name.

Thinking about it, the quoted part means that own builds won't have MapView support though. Maybe we should indeed pull it from a gradle properties file and make the full build fail if it's not defined?

Reply:
> Maybe we should indeed pull it from a gradle properties file and make the full build fail if it's not defined?

The gradle file must be encrypted, right? In this case mapview support would also be absend in own builds if you dont have an own api key. This would require for everyone who want to test something with oh cloud (no foss build possible) to get a api key.
I'd rather have no mapview support in own full builds instead of having a failing build.

Reply:
> The gradle file must be encrypted, right?

No, I would keep it local, that is, not checked in. The Travis environment for doing the Play Store builds would need to have it deployed somehow.

> I'd rather have no mapview support in own full builds instead of having a failing build.

OK. My line of thinking was people not realizing they need to have a local properties file and then wondering why MapView isn't working.